<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuonCollectionSettingsForm</class>
 <widget class="QWidget" name="MuonCollectionSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Muon line options</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="VP1MaterialButton" name="pushButton_matButton">
          <property name="text">
           <string>Material</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Track line widths:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_trackWidth">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="checkBox_trackTubes">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Display lines with cylinders</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_trackTubesRadiusMM">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="prefix">
               <string>r = </string>
              </property>
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_tracksUseBaseLightModel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Use a less advanced light model for the tracks. This might lead to more clearly visible tracks (but transparency will not be taken into account)</string>
          </property>
          <property name="text">
           <string>Use base lighting</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_hideactualpaths">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Do not display the actual tracks. Projections and other track details (measurements, errors, ...) will still be shown.</string>
          </property>
          <property name="text">
           <string>Hide line</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="VP1EtaPhiCutWidget" name="etaPhiCutWidget_2" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="cuts" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_cuts_momentum">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Momentum cuts</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QGridLayout" name="_5">
             <property name="verticalSpacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_cut_minpt">
               <property name="text">
                <string>Min</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_cut_minpt_gev">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> GeV</string>
               </property>
               <property name="maximum">
                <double>999.990000000000009</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox_cut_maxpt">
               <property name="text">
                <string>Max</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_cut_maxpt_gev">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> GeV</string>
               </property>
               <property name="maximum">
                <double>999.990000000000009</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_momtype">
             <item>
              <property name="text">
               <string>Pt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="VP1EtaPhiCutWidget" name="etaPhiCutWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_cuts_momentum_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Quality cuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Minimum quality </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_minimumQuality">
          <property name="toolTip">
           <string>Minimum quality level shown. 'Very Loose' is equivalent to no cut.</string>
          </property>
          <item>
           <property name="text">
            <string>Very Loose</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Loose</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tight</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Associated objects</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_showTrackParticles">
          <property name="toolTip">
           <string>Show the trackparticles associated with this muon. You can chose which to show with the pulldown.</string>
          </property>
          <property name="text">
           <string>TrackParticles</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_shownTrackParticles">
          <item>
           <property name="text">
            <string>Primary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Combined</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Muon Standalone</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Muon Extrapolated</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_showCaloClusters">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Calo clusters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_showSegments">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Segments</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1MaterialButton</class>
   <extends>QPushButton</extends>
   <header>VP1Base/VP1MaterialButton.h</header>
  </customwidget>
  <customwidget>
   <class>VP1EtaPhiCutWidget</class>
   <extends>QWidget</extends>
   <header>VP1Base/VP1EtaPhiCutWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_showTrackParticles</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_shownTrackParticles</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
